precompiles/precompile.go

```diff
@@ -536,6 +535,11 @@ func Precompiles() map[addr]ArbosPrecompile {
 		}
 	}
 
+	ArbOwnerPublic := insert(MakePrecompile(templates.ArbOwnerPublicMetaData, &ArbOwnerPublic{Address: hex("6b")}))
+	getter := ArbOwnerPublic.methodsByName["GetInfraFeeAccount"]
+	getter.arbosVersion = 5
+	ArbOwnerPublic.methodsByName["GetInfraFeeAccount"] = getter
```

Comment:
A simpler way to write this (and the assignments below) is to take a reference with `getter := &ArbOwnerPublic.methodsByName["GetInfraFeeAccount"]`